docs/markdown/spline_intro.md

```diff
@@ -183,10 +187,10 @@ Subsequently, the degree of the basis function can be raised bit by bit using
 until the desired degree has been reached.
 
 
-####    Properties of B-spline basis functions:
+#### Properties of B-spline basis functions:
 -  constitute a partition of unity :math:`\sum_{i=1}^{n}N_{i,p}(u)=1`
 -  non-negative
--  continuously differentiable within knot spans, :math:`p\cdots` times differentiable across knots (multiplicity of knot)
+-  continuously differentiable within knot spans, :math:`p` times differentiable across knots (multiplicity of knot)
```

Comment:
Not sure why this is missing here, but could you add that the basis functions attain exactly one maximum on the interval [t_p+1, t_n+1] and that the same properties also hold for their rational counterparts.

Reply:
(except zero-degree)